src/test/java/teammates/test/cases/webapi/GetAuthInfoActionTest.java

```diff
@@ -126,8 +134,38 @@ protected void testExecute() {
                 Const.ParamsNames.USER_ID, "idOfAnotherInstructor",
         }));
 
-        // TODO test CSRF token cookies
+        ______TS("Test adding CSRF token cookies");
+
+        loginAsInstructor("idOfInstructor1OfCourse1");
+
+        MockHttpServletRequest mockRequest = new MockHttpServletRequest(getRequestMethod(),
+                Const.ResourceURIs.URI_PREFIX + getActionUri());
+        mockRequest.addCookie(new Cookie(Const.CsrfConfig.TOKEN_COOKIE_NAME, "some random csrf token"));
+        MockHttpServletResponse mockResponse = new MockHttpServletResponse();
+
+        a = (GetAuthInfoAction) new ActionFactory().getAction(mockRequest, getRequestMethod(), mockResponse);
```

Comment:
Let not break the abstraction from the base action. You can write another `getAction` in BaseActionTest.